### PR TITLE
Adding trouble shooting docs

### LINKED ISSUE #4: [WRITE] Troubleshooting
As discussed during the sync today, the Troubleshooting section (using-radicle/troubleshooting) can be used as a place to store any documentation related to error codes and/or support issues.

I propose we format this section with any detailed error states that can't be communicated within an Upstream notification. These error states can include troubleshooting tips and/or specific directions on how to report the error (i.e. through GitHub issues, Support category on radicle.community)

Not sure who wants to take lead on this, but just assigning to the folks that I've talked this through with. Feel free to rearrange.

**Reminder to add:**

- User workaround for [#1190](https://github.com/radicle-dev/radicle-upstream/pull/1190) @FintanH

### PR DESCRIPTION
**Pull request by FintanH**
Closes #4

[Rendered](https://github.com/radicle-dev/radicle-docs/blob/trouble-shooting/docs/trouble-shooting.md) 

This is the first draft to our trouble shooting section.

I could do with some help sourcing images and other reasons for errors.

I'm not 100% on proxy exit or session error, so I'm hoping @geigerzaehler and @MeBrei could help out there.

Feel free to commit directly :+1:

### CODE REVIEW

File: docs/trouble-shooting.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+> yourself, you will succumb in every battle.”
+>
+> ― Sun Tzu, The Art of War
+
+As developers of Upstream we want you, the user, to have the best experience as
+possible. Also as developers of Upstream, we know that there will be bugs and
+errors. To help consolidate these diametrically opposed pieces of knowledge we
+will try and document any of the errors that we are aware of, and give you
+advice on how you might recover. In the event that this information doesn't
+help, do not hesitate to reach out to [radicle.community](community) — or our
```
Comment (geigerzaehler) [inline docs/trouble-shooting.md:18]:
```suggestion
help, do not hesitate to reach out to [radicle.community][community] — or our
```

The same fix needs to be applied to the other links.

**Review by geigerzaehler — COMMENTED** [review]
I’m going to add a section for “UnexpectedProxyExit”

File: docs/trouble-shooting.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+---
+id: trouble-shooting
+title: Trouble Shooting
```
Comment (rudolfs) [inline docs/trouble-shooting.md:3]: "Troubleshooting" is one word.

File: website/sidebars.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
       "understanding-radicle/glossary",
       "understanding-radicle/faq"
     ],
-    "Building on Radicle": ["building-on-radicle/documentation"]
+    "Building on Radicle": ["building-on-radicle/documentation"],
+    "Trouble Shooting": ["trouble-shooting"]
```
Comment (rudolfs) [inline website/sidebars.json:20]:
```suggestion
    "Troubleshooting": ["trouble-shooting"]
```

File: docs/trouble-shooting.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+100644 blob e3f6e6bd25955802060698f5b5449c874969aa71    refs
+
+$ git cat-file -p e3f6e6bd25955802060698f5b5449c874969aa71
+{"refs":{"heads":{},"remotes":{}},"signature":"hyydp6mkwkrx1cuz8i4pg3wfojujxaa3s81higa846msaroegcdspp3qe358wmpa3f85s6zmyha5zfgtqnkszwcc6a3k7pfd35fnceoec"}
+```
+
+If you report this JSON payload and the offending Device ID, we can take a look
+and see if we notice anything out of the ordinary.
+
+## UnexpectedBackendExit
```
Comment (geigerzaehler) [inline docs/trouble-shooting.md:144]: I’ve renamed “proxy” to “backend”. I think the latter is much easier to understand for users. I’m also thinking about renaming this error to `BackendTerminated` which would make it more aligned with the usual terminology. Any thoughts

Of course this needs to be followed up with a change to the app.

**Comment by rudolfs** [comment]
Added the error notification screenshot.

File: docs/trouble-shooting.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+As developers of Upstream we want you, the user, to have the best experience as
+possible. Also as developers of Upstream, we know that there will be bugs and
+errors. To help consolidate these diametrically opposed pieces of knowledge we
+will try and document any of the errors that we are aware of, and give you
+advice on how you might recover. In the event that this information doesn't
+help, do not hesitate to reach out to [radicle.community](community) — or our
+[GitHub issues](github-issues) if you are comfortable with opening a detailed
+issue.
+
+For our known failures, we categorise them with a camel-case code word that
```
Comment (xla) [inline docs/trouble-shooting.md:22]:
```suggestion
For our known failures, we categorise them with a PascalCase code word that
```

Either that or upper camel case.

File: docs/trouble-shooting.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+This is catch all error. As we categorise our errors more & more — and even
+better, eliminate them! — then we should see less & less of this particular
+case.
+
+### Action
+
+Please report these errors to [GitHub](github-issues) with as much detail as
+possible — checking first that it's not already reported by another user.
+
+[community]: https://radicle.community/
```
Comment (xla) [inline docs/trouble-shooting.md:174]:
```suggestion
[community]: https://matrix.radicle.community/
```

**Review by xla — CHANGES_REQUESTED** [review]
Love a quote, but the war theme might be questionable.

File: website/sidebars.json (hunk 3 above)
Comment (rudolfs) [reply website/sidebars.json:20]: On a second thought: https://github.com/radicle-dev/radicle-docs/pull/21/files#r522813404.

File: docs/trouble-shooting.md (hunk 1 above)
Comment (rudolfs) [reply docs/trouble-shooting.md]: https://github.com/radicle-dev/radicle-docs/pull/21/commits/e7b7b2f4163fcb8932e68db1dfc256b6a30699da

**Comment by FintanH** [comment]
> Love a quote, but the war theme might be questionable.

But is the Art of War really about war? _Really?_

File: website/sidebars.json (hunk 3 above)
Comment (rudolfs) [reply website/sidebars.json]: Fixed in: https://github.com/radicle-dev/radicle-docs/pull/21/commits/4f69e2a10e0edd7040fe7f3a72590f93a7a4c21e.

File: docs/trouble-shooting.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
+---
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply docs/trouble-shooting.md]: Fixed in: https://github.com/radicle-dev/radicle-docs/pull/21/commits/4f69e2a10e0edd7040fe7f3a72590f93a7a4c21e.

**Review by rudolfs — APPROVED** [review]

**Pull request merged**